mentoring/new-contributor-orientation/new-contributor-resources.md

```diff
@@ -1,8 +1,38 @@
-Join the Kubernetes Developer Google Group (k-dev): https://groups.google.com/a/kubernetes.io/g/dev
-Check out [kubernetes/community](https://github.com/kubernetes/community) for an overview of our community.
-Follow the New Contributor Guide (k8s.dev/guide) to get familiar with various contributor tools and processes.
-The First Contribution Guide(k8s.dev/guide/first-contribution/) will walk you through the process for your first Pull Request (PR)
-Check out the Resources List at https://bit.ly/kubernetes-resources
-Come join us in the Kubernetes slack: https://www.slack.k8s.io 
-* Join the #kubernetes-new-contributors channel
-* Join the channels of 1 or more SIGs you’re interested in
\ No newline at end of file
+# Kubernetes New Contributors Resource
+
+Welcome to the Kubernetes community! This guide will help you get started with contributing effectively.
+
+## 1. Join the Community
+
+- **Kubernetes Developer Google Group (k-dev):**  
+  Stay updated by joining the [k-dev Google Group](https://groups.google.com/a/kubernetes.io/g/dev).
+
+- **Community Overview:**  
+  Explore the [Kubernetes Community Repository](https://github.com/kubernetes/community) to learn about our structure and initiatives.
+
+- **Slack Communication:**  
+  Connect with contributors on [Kubernetes Slack](https://kubernetes.slack.com/):  
+  - Join the `#kubernetes-new-contributors` channel to introduce yourself.  
+  - Participate in channels of SIGs you're interested in.
+
+## 2. Learn and Get Familiar
+
+- **New Contributor Guide:**  
+  Follow the [New Contributor Guide](https://k8s.dev/guide) to understand our tools and workflows.
+
+- **First Contribution Guide:**  
+  The [First Contribution Guide](https://www.kubernetes.dev/docs/guide/first-contribution/) walks you through submitting your first Pull Request (PR).
+- **New Contributor Course:**  
+  Check out the [New Contributon Course](https://www.kubernetes.dev/docs/onboarding/) 
+  for a high-level overview of the contribution process.
+
+
+
+## 3. Explore Resources
+
+- **Resource List:**  
+  Access a curated [list of resources](https://bit.ly/kubernetes-resources) for contributors.
```

Comment:
This is not a blocker for this PR, but I'd say that we really need another PR to integrate the resource list with the main docs instead of linking to a Gist written by Dims 3 years ago.  I'll open an issue.